backend/src/main/kotlin/blue/mild/covid/vaxx/setup/ConfigurationDependencyInjection.kt

```diff
@@ -47,8 +47,8 @@ fun DI.MainBuilder.bindConfiguration() {
             apiKey = requireEnv(EnvVariables.MAIL_JET_API_KEY),
             apiSecret = requireEnv(EnvVariables.MAIL_JET_API_SECRET),
             emailFrom = getEnvOrLogDefault(EnvVariables.MAIL_ADDRESS_FROM, "ockovani@mild.blue"),
-            nameFrom = getEnvOrLogDefault(EnvVariables.NAME_FROM, "Registrace Očkování"),
-            subject = getEnvOrLogDefault(EnvVariables.SUBJECT, "Detaily k registraci na Očkování")
+            nameFrom = getEnvOrLogDefault(EnvVariables.MAIL_FROM, "Registrace Očkování"),
+            subject = getEnvOrLogDefault(EnvVariables.MAIL_SUBJECT, "Detaily k registraci na Očkování")
```

Comment:
nechapu to velke O, to by melo byt velke i Registraci

Reply:
cc @kubantjan 😃  tak jak to udelame? s malym?

Reply:
Asi jo :D